packages/gravity-vue/package.json

```
@@ -34,6 +34,7 @@
   "scripts": {
     "build": "vite build",
     "test": "vitest --run",
+    "test:only": "vitest -w -t ",
```

лучше так, дополнительные аргументы не нужны
```suggestion
    "test:vitest": "vitest",
```